desktop-respons.css

```diff
@@ -0,0 +1,241 @@
+@media screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+  }
+
+  .header-at-top {
+    background-image: url(./img/Header\ bg.svg);
+    background-size: cover;
+  }
+
+  .menu-phone {
+    display: none;
+  }
+
+  .header-containers {
+    background-color: #fff;
+  }
+
+  .logo {
+    padding-left: 143px;
+    color: #6070ff;
+  }
+
+  .navbar-items {
+    padding-right: 142px;
+  }
+
+  .navbar-items a {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #6070ff;
+    text-decoration: none;
+    padding: 0 12px;
+  }
+
+  .primary-text,
+  .social,
+  .social-media-icons {
+    padding-left: 260px;
+  }
+
+  .text-about {
+    padding: 0 260px;
+  }
+
+  .work-container {
+    display: grid;
+    column-gap: 0;
+    row-gap: 88px;
+  }
+
+  .box {
+    display: flex;
+    flex-direction: row;
+    padding: 24px;
+    width: 60%;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    height: fit-content;
+  }
```

Comment:
- Hey, please, kindly ensure you've reversed the direction of the second and fourth cards, you can check the Figma design to see it. You can use the `flex-direction: row-reverse` to fix this.